CHANGES.md

```diff
@@ -1,5 +1,10 @@
 # Changelist of gmig releases
 
+## v1.6.0
+
+- add options to new to set the commands for the do,undo,view section directly
```

Comment:
Can you also update the README.md to reflect this new feature for the "new" command